pkg/uefi/meregion.go

```diff
@@ -48,7 +48,8 @@ type MEPartitionEntry struct {
 	Owner    [4]byte
 	Offset   uint32
 	Length   uint32
-	Reserved [4]uint32
+	Reserved [3]uint32
```

Comment:
You can name this _ if you want. Do you need the Reserved here for some reason? Did you want to save the value of Reserved bits?

Reply:
Would advise preserving the value of Reserved bits.

Reply:
Currently it's only used for parsing and the full buffer is preserved, but if for some reasons we want to go in more detailed analysis and modification, it's better to keep the content of those fields. Especially since they mostly are not reserved anymore but the documentation I used (older versions of ME) listed them as reserved.